src/input/speaker_notes.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(Debug)]
+#[repr(C)]
+pub enum SpeakerNotesCommand {
```

Comment:
Maybe there could be an `Exit` command sent when presenterm exits? That way they both would close simultaneously.

Reply:
Good idea.

Reply:
Done.